tests/integration/conftest.py

```diff
@@ -856,6 +785,70 @@ def configure_local_internal_options_module(request):
     conf.set_local_internal_options_dict(backup_local_internal_options)
 
 
+@pytest.fixture(scope='function')
+def daemons_handler_function(request):
+    """Handler of Wazuh daemons.
+
+    Args:
+        request (fixture): Provide information on the executing test function.
+    """
+    try:
+        daemons_handler_configuration = getattr(request.module, 'daemons_handler_configuration')
+
+    except AttributeError as daemon_configuration_not_set:
+        logger.error('daemons_handler_configuration is not set')
+        raise daemon_configuration_not_set
+
+    start_daemons(daemons_handler_configuration['function'])
+
+    yield
+
+    stop_daemons(daemons_handler_configuration['function'])
+
+
+@pytest.fixture(scope='module')
+def daemons_handler_module(request):
```

Comment:
This will break all the current test that uses `daemon_handler`. Please, maintain the same name or change all the tests accordingly.

Reply:
Done in 64a89f7